pkgs/applications/video/kodi-cli/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, makeWrapper, pkgs, curl, bash, jq, youtube-dl, gnome3, ...}:
+
+let
+  pkgName = "kodi-cli";
+  version = "1.1.1";
+in
+
+stdenv.mkDerivation rec {
+  name = pkgName + "-" + version;
+  src = fetchFromGitHub {
+	owner = "nawar";
+	repo = pkgName;
+	rev = version;
+	sha256 = "0f9wdq2fg8hlpk3qbjfkb3imprxkvdrhxfkcvr3dwfma0j2yfwam";
+  };
+
+
+  buildInputs = [ makeWrapper ];
+
+  installPhase = ''
+	mkdir -p "$out"/bin
+	cp -a kodi-cli "$out"/bin
+	wrapProgram "$out/bin/kodi-cli" --prefix PATH : "${stdenv.lib.makeBinPath [ curl bash ]}"
+	cp -a playlist_to_kodi "$out"/bin
+	wrapProgram "$out/bin/playlist_to_kodi" --prefix PATH : "${stdenv.lib.makeBinPath [ curl bash gnome3.zenity jq youtube-dl]}"
+  '';
+
+  meta = with stdenv.lib; {
+    inherit (src.meta) homepage;
+    description = "Kodi/XBMC bash script to send Kodi commands using JSON RPC. It also allows sending YouTube videos to Kodi";
+    license = licenses.gpl2;
+    platforms = platforms.all;
+    maintainers = [ maintainers.pstn ];
```

Comment:
You'll need to add yourself to the https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix

Reply:
I'm already in there somewhere. Wouldn't @GrahamcOfBorg complain otherwise btw?